nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -129,7 +129,8 @@ define([
                 bus: runtime.bus(),
             }),
             typesToFiles = setupFileData(options.importData),
-            { fileTypesDisplay } = BulkImportUtil.generateFileTypeMappings(typesToFiles),
+            { fileTypesDisplay, fileTypeMapping } =
+                BulkImportUtil.generateFileTypeMappings(typesToFiles),
```

Comment:
pass this data on to the different panels (configure, results, job status, etc.)